includes/Story_Renderer/HTML.php

```diff
@@ -191,6 +253,49 @@ protected function replace_html_head( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Remove <noscript> AMP boilerplate fragment from the existing markup.
+	 *
+	 * The libxml extension prior to version 2.8.0 has issues parsing <noscript> tags in the <head>.
+	 * That's why we temporarily remove this fragment and re-add it later.
+	 *
+	 * @see https://github.com/ampproject/amp-wp/pull/5097
+	 * @see http://xmlsoft.org/news.html
+	 *
+	 * @param string $content Story markup.
+	 *
+	 * @return string Filtered content.
+	 */
+	protected function remove_noscript_amp_boilerplate( $content ) {
+		return str_replace( '<noscript><style amp-boilerplate="">body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>', '', $content );
```

Comment:
Minor thing: the string appears here and in `add_noscript_amp_boilerplate`. It could be put in a constant since the string literal is the same.